<ui version="4.0" >
 <class>StyleSheetEditor</class>
 <widget class="QWidget" name="StyleSheetEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="1" colspan="2" >
    <widget class="QComboBox" name="layoutCombo" >
     <item>
      <property name="text" >
       <string>boringdefault</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>waterfall</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="layoutLabel" >
     <property name="text" >
      <string>Layout</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2" >
    <widget class="QComboBox" name="styleSheetCombo" >
     <item>
      <property name="text" >
       <string>boringdefault</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>coffee</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_8" >
     <property name="text" >
      <string>&lt;nobr>Style Sheet&lt;/nobr></string>
     </property>
     <property name="textFormat" >
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3" >
    <widget class="QTextEdit" name="styleTextEdit" />
   </item>
   <item row="4" column="2" >
    <widget class="QPushButton" name="previewButton" >
     <property name="text" >
      <string>&amp;Preview</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label_7" >
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:MS Shell Dlg 2; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Style&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="textFormat" >
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2" >
    <widget class="QComboBox" name="styleCombo" >
     <item>
      <property name="text" >
       <string>CDE</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Cleanlooks</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Motif</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Plastique</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Windows</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>WindowsXP</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
